src/app/shared/backup/backup.service.ts

```diff
@@ -16,12 +14,43 @@ export class BackupService extends EntityService {
       this.BACKUP_BASEURL = customEndpointService.getUri('osb-backup-manager');
   }
 
-  public getServiceInstanceId(): string {
-    return serviceInstanceId;
+  public getServiceInstance(): string {
+    return environment.serviceInstance;
   }
 
-  public loadAll(entityRel: string): Observable<{} | any> {
-    return this.all(this.BACKUP_BASEURL + '/' + entityRel + '/byInstance/' + serviceInstanceId + '?' + this.SORT_CONFIG);
+  private pagingAndSortingHandler(pagingAndSorting: any): string {
+    var resultString: string = "";
+
+    if (pagingAndSorting == null || pagingAndSorting.length == 0)
+      return resultString;
+
+    resultString += "?size=" + pagingAndSorting.pageSize;
+    resultString += "&page=" + (pagingAndSorting.page - 1);
+
+    return resultString;
+  }
+
+  private filterHandler(filterQuery: any): string {
+    var resultString: string = "";
+
+    if (filterQuery == null)
+      return resultString;
+
+    Object.keys(filterQuery).forEach(key => {
+      resultString += "&" + key + "=" + filterQuery[key];
+    });
+
+    return resultString;
+  }
+
+  public loadAll(entityRel: string, pagingAndSorting?: any): Observable<{} | any> {
```

Comment:
I removed `<{} | any>` and replaced it with `<any>`. The other point is not solvable in this context, because the common subset of all objects that are retrieved is object.